e2e-tests/mongod-major-upgrade-sharded/run

```diff
@@ -129,6 +129,12 @@ function main() {
 		run_mongos 'use myApp\n db.test.insert({ x: 100600 })' "myApp:myPass@${cluster}-mongos.${namespace}"
 		run_restore "${backup_name_minio}"
 		wait_restore "${backup_name_minio}" "${cluster}"
+
+		if [[ ${version} =~ 5.0 ]]; then
```

Comment:
To be honest to me this looks like a dirty hack, we should escalate this to PBM team and comment out backup/restore from this test or just make the backup/restore on the same major version and not do downgrade with restore.